lib/aptible/cli/helpers/database.rb

```diff
@@ -90,8 +175,19 @@ def replicate_database(source, dest_handle, options)
 
         # Creates a local tunnel and yields the helper
 
-        def with_local_tunnel(credential, port = 0)
-          op = credential.create_operation!(type: 'tunnel', status: 'succeeded')
+        def with_local_tunnel(credential, port = 0, target_account = nil)
```

Comment:
there is a contract change here, we can breka this as well into a fork here. I chose to reuse the same function with an optional parameter and a fork in the parameters of the operation